translations/fr.json

```diff
@@ -1,339 +1,330 @@
   "tribes.aquarion": "Aquarion",
-  "tribes.aquarion.info": "Des océans profonds, une ancienne civilisation émerge ! Leur isolement extrême leurs a donné des capacités aquatiques spéciales qui sont inconnues des autres tribus humaines.\n\nLes {0} ont un arbre de recherche un peu différent et ont accès à une unité amphibie unique, des tortues de mer qu'aucune autre tribu ne peut entraîner.",
+  "tribes.aquarion.info": "Des océans profonds une civilisation perdue depuis longtemps ! Leur isolement extrême leur a donné des capacités aquatiques spéciales qui sont inconnues des autres tribus plus humaines. \n \n {0} ont des technologies légèrement différentes et ont accès à l'unité unique de tortues amphibies qu'aucune autre tribu ne peut entraîner. ",
   "tribes.quetzali": "Quetzali",
-  "tribes.quetzali.info": "La tribu {0} adore les Divinités Oiseaux dde la Terre Rouge et vivent en harmonie avec la symétrie naturelle de leurs jungles cubistes. On les voit souvent chevaucher d’énormes oiseaux incapables de voler. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
+  "tribes.quetzali.info": "La {0} tribu adore les dieux oiseaux du sol rouge et vit en harmonie avec la symétrie naturelle de leurs jungles cubistes. On les voit souvent chevaucher d’énormes oiseaux incapables de voler. \n \n {0 } démarre le jeu avec la technologie '{1}'. ",
   "tribes.elyrion": "∑∫ỹriȱŋ",
-  "tribes.elyrion.info": "Les mystérieux {0} défendent leur forêt avec une puissante magie et la férocité du feu des dragons ! \n\nLeur nature est sacrée, il ne chasse donc pas d'animaux et ne coupe pas d'arbres. Au lieu de cela, ils commencent la partie avec une technologie unique qui leur permet d'enchanter les animaux ordinaires et les transformer en monstres puissants.",
+  "tribes.elyrion.info": "Les mystérieux {0} défendent leurs maisons boisées avec la magie colorée et la férocité des dragons cracheurs de feu ! \n \nIls considèrent la nature comme un esprit saint et ne peuvent pas chasser les animaux ni abattre les arbres, à la place ils commencent le jeu avec la technologie unique d'enchantement qui peut transformer des animaux ordinaires en monstres puissants.",
   "tribes.yadakk": "Yădakk",
-  "tribes.yadakk.info": "Les {0} commencent comme tribu nomade dans les merveilleuses plaines de Khalee. Désormais, ils sont les commerçants du monde, reliant leur empire à des routes commerciales impressionnantes. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
+  "tribes.yadakk.info": "Les {0} ont commencé comme une tribu nomade dans les belles plaines de Khalee. Maintenant, ils sont les commerçants de la place, reliant leur empire à des routes commerciales impressionnantes. \n \n {0} démarre le jeu avec la technologie '{1}'. ",
```

Comment:
One comments for all this change about descriptions of the tribes. Why do you whant to make all this changes? Most of the time it doesn't respect the original text of the game and it's not beautiful in french... Like the first one "vallées vertes" instead of "vallées verdoyantes"....